features/borrowers_personal_info.feature

```diff
@@ -0,0 +1,435 @@
+               "dob": "29/09/2000",
+               "phone_number": "07507154076"
+           }
+        ]
+        }
+        """
+
+    Then a status code of "201" is returned
+    And a url link to retrieve the deed is returned
+
+Scenario: Create a deed with one Male borrower and one Unknown borrower
+    Given I add the following deed:
+
+    """
+    {
+    "title_number": "DT567568",
+    "borrowers": [
+       {
+           "forename": "Katie",
+           "middle_name": "Elizabeth",
+           "surname": "Smythe",
+           "gender": "U",
+           "address": "1 The High Street Highley CR0 9YH",
+           "dob": "29/09/1965",
+           "phone_number": "07507154077"
+       },
+       {
+           "forename": "Paul",
+           "middle_name": "",
+           "surname": "Smythe",
+           "gender": "M",
+           "address": "1 The High Street Highley PL6 7TG",
+           "dob": "29/09/2000",
+           "phone_number": "07507154076"
+       }
+    ]
+    }
+    """
+
+    Then a status code of "201" is returned
+    And a url link to retrieve the deed is returned
+
+Scenario: Create a deed with one Male borrower and one invalid gender borrower
+    Given I add the following deed:
+
+    """
+        {
+        "title_number": "DT567568",
+        "borrowers": [
+           {
+               "forename": "Katie",
+               "middle_name": "Elizabeth",
+               "surname": "Smythe",
+               "gender": "MFU",
+               "address": "1 The High Street Highley CR0 9YH",
+               "dob": "29/09/1965",
+               "phone_number": "07507154077"
+           },
+           {
+               "forename": "Paul",
+               "middle_name": "",
+               "surname": "Smythe",
+               "gender": "M",
+               "address": "1 The High Street Highley PL6 7TG",
+               "dob": "29/09/2000",
+               "phone_number": "07507154076"
+           }
+        ]
+        }
+        """
+
+        Then a status code of "400" is returned
+        And a message for failure is given "Failed validating 'pattern' in schema['properties']['borrowers']['items']['properties']['gender']:"
+
+Scenario: Create a deed with one Female borrower and one borrower without gender
+    Given I add the following deed:
+
+    """
+    {
+    "title_number": "DT567568",
+    "borrowers": [
+       {
+           "forename": "amanda",
+           "middle_name": "",
+           "surname": "Blah",
+           "gender": "F",
+           "address": "1 High Street cr07YH",
+           "dob": "11/01/2010",
+           "phone_number": "075436789089"
+       },
+       {
+           "forename": "A",
+           "middle_name": "b",
+           "surname": "blah",
+           "address": "blah blah blah pl98ug",
+           "dob": "01/01/2015",
+           "phone_number": "07300000009"
+       }
+    ]
+    }
+    """
+
+    Then a status code of "201" is returned
+    And a url link to retrieve the deed is returned
```

Comment:
General comment for the file, I don't see a scenario to check the rule that a date in the future cannot be specified as the date of birth unless I've missed it completely, might be worth adding this.